tests/test_run_submission.py

```diff
@@ -85,7 +87,51 @@ def test_run_submission(self):
             backward_common_files=[],
             task_list=task_list,
         )
-        submission.run_submission()
+        submission.run_submission(check_interval=2)
+
+        for ii in range(4):
+            self.assertTrue(
+                os.path.isfile(
+                    os.path.join(
+                        self.machine_dict["local_root"], "test_dir/", f"out{ii}.txt"
+                    )
+                )
+            )
+
+    def test_async_run_submission(self):
```

Comment:
Please skip the tests in slurm or PBS if they don't work to ensure the CI passes. Also, could you add some comments on this situation?

Reply:
Ok I will skip the test.
It is weird that, on action triggered by my push and PR, the three slurm test may success. I don't know why and I think it is in the `clean` behavior.